sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/WriteToDestinations.java

```diff
@@ -108,11 +119,40 @@ public KV<ShardedKey<Row>, Row> apply(Row elem) {
                 "Write remaining rows to files",
                 new WriteGroupedRowsToFiles(catalogConfig, dynamicDestinations));
 
+    PCollection<FileWriteResult> writeUngroupedResultPColl = writeUngroupedResult.getWrittenFiles();
```

Comment:
Validate here also: if the input is unbounded then triggering frequency is set. I know you check it above, but also do it here so the local code is validated.

Reply:
Thanks, setting discarding mode

Reply:
Done